eos_data_distribution/ndn/command.py

```diff
@@ -143,7 +133,12 @@ def makeInterest(cmd, local=True, controlParameters={},
         commandInterest.setName(Name("/localhop%s" % cmd))
         # The host is remote, so set a longer timeout.
         commandInterest.setInterestLifetimeMilliseconds(4000.0)
+    if interestLifeTime: commandInterest.setInterestLifetimeMilliseconds(interestLifeTime)
     # NFD only accepts TlvWireFormat packets.
+
+    logger.debug("DUMP: %s", dumpControlParameter(controlParameters))
+    logger.debug("wire encoding: %s", str(controlParameters.wireEncode(TlvWireFormat.get())).encode('quopri'))
```

Comment:
Why did you choose quoted-printable encoding here? UTF-8 would be more compact and readable in the printouts, and is supported by all consoles.